requirements.txt

```diff
@@ -1,2 +1,2 @@
-ops==1.3.0
+ops==2.0.0
```

Comment:
Why the ops change here?

Reply:
TL;DR is that I upgraded the version of ops due to version incompatibilities between ops 1.x and ops 2.x when using the _data_interfaces_ charm library in __data_platform_libs__. If you used _data_interfaces_ with ops 1.x, the charm would fail to start because of a metaclass conflict; `ops.framework.Object` would conflict with `abc.ABCMeta`. The metaclass for `ops.framework.Object` was removed in ops 2.x, so _data_interfaces_ would work no problem. Therefore, I decided to update to ops 2.x rather than maintain our own fork of _data_interfaces_.

You can check out issue https://github.com/canonical/data-platform-libs/issues/45 for more context and what we decided on the solution to be.